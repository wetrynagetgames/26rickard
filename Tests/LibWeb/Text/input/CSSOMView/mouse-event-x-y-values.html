<style>
    div {
        width: 100px;
        height: 100px;
        background-color: gray;
        transform: rotate(45deg);
    }
    body {
        height: 200vh;
    }
</style>
<div onClick="
    println(event.x);
    println(event.y);
    println(event.screenX);
    println(event.screenY);
    println(event.clientX);
    println(event.clientY);
    println(event.pageX);
    println(event.pageY);
    println(event.offsetX);
    println(event.offsetY);
"></div>

<script src="../include.js"></script>

<script>
    test(() => {
        window.scrollBy(0, 10)
        internals.click(50, 50);
    });
</script>
