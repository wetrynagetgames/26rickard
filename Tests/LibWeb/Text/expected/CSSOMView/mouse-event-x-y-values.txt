50
50
50
50
50
50
50
60
45.75
57.078125
